### ISSUE TITLE
Search / replace sometimes seems to break

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#2483]
It still highlights words but when you press enter it doesn't jump to the word you search for / doesn't replace all of the searched words with the replace word

I'm talking about both search and search and replace both seemingly breaking sometimes

**Comment by erikangelin-king** [comment]
Can you provide a repro case? You're on windows right?

**Comment by subsoap** [comment]
Yes, Windows 10. If I can figure out how to make it consistently break I will. Usually I notice when searching that it no longer would skip to the next work (but does highlight the searched words so at least I can scroll down and manually look for them). That happens often. The replace also breaks, it just refuses to replace words no matter what when that breaks.
